src/app/components/layout/header/top-bar/top-bar.component.ts

```diff
@@ -54,13 +53,13 @@ export class TopBarComponent implements OnInit, OnDestroy {
     );
 
     this.subscription.add(
-      this.balanceService.totalBalance.subscribe((balance: TotalBalance) => {
-        if (balance && !this.balanceObtained) {
-          this.balanceObtained = true;
+      this.balanceService.totalBalance.subscribe(balance => {
+        if (balance) {
+          this.updateTimeSinceLastBalanceUpdate();
+          this.balanceObtained = balance.state === BalanceStates.Obtained;
```

Comment:
when the balance is being updated, obtained is false, which makes the update animation hidden